1.1-alphafold-quick-start-multimer.ipynb

```diff
@@ -114,7 +114,7 @@
     "FILESTORE_SHARE = '/datasets'\n",
     "FILESTORE_MOUNT_PATH = '/mnt/nfs/alphafold'\n",
     "MODEL_PARAMS = f'gs://{BUCKET_NAME}'\n",
-    "IMAGE_URI = f'gcr.io/{PROJECT_ID}/alphafold-components'"
+    "IMAGE_URI = f'{REGION}-docker.pkg.dev/{PROJECT_ID}/alpha-kfp/alphafold-components'"
```

Comment:
same in here, because 'alpha-kfp' is something use will have been entered from ARTIFACT_REGISTRY_REPO_NAME, before they run terraform. Which could be different value than 'alpha-kfp'